external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -271,10 +272,14 @@ def save_jacobians(std_jacobians, dir_, filename="jacobians.npz"):
 
 def main(config: TrainConfig, seed: int = 0):
     logging.basicConfig(level=getattr(logging, config.log_level))
+    logging.getLogger("fv3fit.keras.callbacks").setLevel(logging.DEBUG)
     set_random_seed(seed)
 
     # callbacks that are always active
-    callbacks = [tf.keras.callbacks.TerminateOnNaN()]
+    callbacks = [
+        tf.keras.callbacks.TerminateOnNaN(),
+        GarbageCollectionCallback(batch_frequency=100),
```

Comment:
I haven't seen garbage collection affect memory usage in my testing, so I'm inclined just to remove the usage.

Reply:
Sure. I'll revert this.